tests/test_generate_multiple_forecasts.py

```diff
@@ -0,0 +1,37 @@
+import os
+import pandas as pd
+import subprocess
+import random
+
+
+def test_generate_forecasts():
+    # Generate 100 random PV IDs between 1 and 50000
+    pv_ids = [random.randint(1, 50000) for _ in range(100)]
```

Comment:
How are you passing in the metadata such as latitude, longitude to be used in the function? I would create 2/3 dummy sites to use instead.